### ISSUE TITLE
SET wrong path to DHCP server

### ISSUE THREAD

**Issue by gozes** [ArchAssault-Project/archassault#56]
when using the wireless attack vectors SET complains about a missing isc-dhcp-server pagakage and even with the dhcp package installed so my guess is that there is a missing ln -s somewhere

**Comment by archey** [comment]
Yeah set needs alot of love as its really written for debian, we will have to fork it and fix it.
